src/CMakeLists.txt

```diff
@@ -35,3 +35,12 @@ endif()
 
 add_library(cucumber-cpp-nomain STATIC ${CUKE_SOURCES})
 add_library(cucumber-cpp STATIC ${CUKE_SOURCES} main.cpp)
+
+if(MINGW)
```

Comment:
The same should work for MSVC too (it does at work). So I suggest you try with `if(WIN32)` instead.